estate/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
+    'name': 'Real Estate',
+    'version': '18.0',
```

Comment:
```suggestion
    'version': '1.0',
```
This is the version of the package, not the version of Odoo.